devicetypes/ecolink/ecolink-dwzb1-eco-contact-sensor.src/ecolink-dwzb1-eco-contact-sensor.groovy

```diff
@@ -0,0 +1,243 @@
+/*
+ *  Copyright 2016 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+
+metadata {
+	definition(name: "Ecolink DWZB1-ECO Contact Sensor", namespace: "Ecolink", author: "Ecolink") {
+		capability "Contact Sensor"
+		capability "Configuration"
+		capability "Battery"
+		capability "Temperature Measurement"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Sensor"
+
+		command "enrollResponse"
+        
+        attribute "tamper", "enum", ["detected", "clear"]
+
+		//fingerprints
+        fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,0020,0500,0B05", outClusters: "0019", manufacturer: "Ecolink", model: "DWZB1-ECO"//, deviceJoinName: "Ecolink Door/Window"
```

Comment:
@marchbold42 I think there's no need for whole new DTH. Could you add fingerprint of your device to `smartsense-open-closed-sensor.groovy`? It should cover its whole functionality.

Reply:
Using an existing DTH is certainly preferable if possible.  It looks like this has some special tamper handling on the ias zone message.  But there is already ecolink related configuring in the other one and it looks like there should still be some ability to reuse that for this device as well.